### DISCUSSION TITLE
ContentScrollable Default might be not working as expected?

### DISCUSSION THREAD

**Discussion by imgrammy** [Blazored/Modal#317] [General]
The browser scroller doesn't appear, then the content gets stuck.

![image](https://user-images.githubusercontent.com/1329782/119186300-78461e80-baaa-11eb-8523-1fb6fb1b742a.png)

**Comment by chrissainty** [comment]
Did you click the ContentScrollable Enabled button or the ContentScrollable Default? As the default is to not enable scrollable behaviour. Checking the sample application running [here](https://blazored.github.io/Modal/scrollablecontent) everything is working fine.
